src/Cdn/Cdn/Common/AzureCdnCmdletBase.cs

```diff
@@ -18,12 +18,14 @@
 using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
 using Microsoft.Azure.Commands.ResourceManager.Common;
 using Microsoft.Azure.Management.Cdn;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Cdn.Common
 {
     /// <summary>
     /// Base class of Azure Cdn Cmdlet.
     /// </summary>
+    [GenericBreakingChange("This command will be redesigned", "2.0.0")]
```

Comment:
Hi @ChenglongLiu , could you please let me what exact breaking change will be done? Here is the process for breaking change: https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azps_docs/breakingchange_release_process

Reply:
@BethanyZhou According to the meeting with powershell team, powershell based on c# SDK should be deprecated. All the CDN powershell command should generated from Swagger dicrectly, which will cause all commads different from the pervious verison.